content/en/docs/contribute/localization.md

```diff
@@ -205,14 +205,18 @@ To ensure accuracy in grammar and meaning, members of your localization team sho
 
 ### Source files
 
-Localizations must be based on the English files from the most recent release, {{< latest-version >}}.
+Localizations must be based on the English files from your target version.
 
-To find source files for the most recent release:
+To find source files for your target version:
 
 1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+2. Select branch for your target version from followings.
 
-The latest version is {{< latest-version >}}, so the most recent release branch is [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
```

Comment:
We wil want to keep this.
If `release-1.20` is not there yet, let's create it.

Reply:
Hello @tengqm :) 
As you know, `master` branch includes the latest documents of the latest K8s release. (currently, v1.20)
I think we are using `release-x.xx` branch for maintaining k8s.io documents for previous K8s version.

So, I am not sure representing `the most recent release branch is release-x.xx` is appropriate.

 if we create `release-1.20` branch, we might need to keep updating that branch as well according to the changes in the `master` branch. 
I think release team will create `release-1.20` branch just before the K8s v1.21 release.

In case of Korea 10n team, our translation target is `master` branch (currently targeted K8s release version is v1,20),

Reply:
Valid question. If I'm understanding this correctly, we are always treating the `master` branch as the least release. So, for release `1.20`, the `release-1.20` branch is meaningless.
If this understanding is correct, we may want to explicitly spell it out as a replacement to line 215. WDYT?

Reply:
Thank you for reviewing. Please check recent commit.